app.rb

```diff
@@ -0,0 +1,18 @@
+require 'rubygems'
+require 'bundler'
+
+Bundler.require
+
+require './app/base'
+
+# The global module for Onyx, all Onyx related code will be in the Onyx module
+# @author Noah Huppert <developer.noah@gmail.com>
+module Onyx
+	# The main entrypoint for Onyx
+	# @author Noah Huppert <developer.noah@gmail.com>
+	class App < Base
+		get '/' do
+			return 'Root'

```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.